demo/index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Iron Label Demo</title>
```

Comment:
We've been trying to keep our `meta name="viewport"` tags in a row: https://github.com/PolymerElements/paper-button/blob/master/demo/index.html#L17